.travis.yml

```diff
@@ -4,5 +4,6 @@ node_js:
 - '6'
 - '8'
 - '10'
+- '12'
```

Comment:
would you mind to send a separate PR for updating the travis config?

Reply:
done: https://github.com/pinojs/pino-caller/pull/19